packages/react/radio-group/src/RadioGroup.tsx

```diff
@@ -92,47 +102,37 @@ RadioGroup.displayName = RADIO_GROUP_NAME;
 
 const ITEM_NAME = 'RadioGroupItem';
 
-type RadioGroupItemOwnProps = Polymorphic.OwnProps<typeof RadioGroupItemImpl>;
+type RadioGroupItemOwnProps = Polymorphic.Merge<
+  Polymorphic.Merge<
+    Omit<Polymorphic.OwnProps<typeof RovingFocusItem>, 'focusable' | 'active'>,
+    Polymorphic.OwnProps<typeof Radio>
+  >,
+  { value: string; name?: never }
```

Comment:
Fixed the remove `name` prop as it's a DOM attribute it would get re-added otherwise.